D/DSGE/Deps.toml

```diff
@@ -55,3 +55,6 @@ SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsPlots = "f3b207a7-027a-5e70-b257-86293d7955fd"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+["0.5.1-0"]
+InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
```

Comment:
Was this intentional? I am guessing no since you only use it withing tests.

Reply:
Maybe have a look at https://julialang.github.io/Pkg.jl/v1/creating-packages/#Test-specific-dependencies-1

Reply:
ah I was not aware of this feature, I'll make the change and re-submit this PR.

Reply:
Just retrigger the bot and this PR will be updated.

Reply:
Yep! Retrigged.

Reply:
See https://github.com/FRBNY-DSGE/DSGE.jl/commit/7f7ff1c283060f34f09d63745fc861712d73d92b#r33855882

Reply:
I'll retrigger the bot once our CI passes, just so I don't keep bumping you with commits that need to be revised.